talent/onboarding/README.md

```diff
@@ -39,6 +39,10 @@ During your first week you are going to have Welcome Talks in order to get to kn
 * Welcome talk with the Head of Product and Data Intelligence
 * Welcome talk the Sales team
 
+### Welcome Breakfast & Lunch
+
+- If your first day at source{d} is in the office, we will organise a welcome breakfast with various members of the team so that you can get to know them, and will also have a member of the team take you out for lunch too.
```

Comment:
Only one member to have lunch with the new guy? Why not a team event?

Reply:
In my experience, a whole team lunch and welcome breakfast might be overwhelming.... also gives them a chance one on one in a relaxed atmosphere to ask questions etc....but I'm open to the idea!

Reply:
(I'd purpose at least 2 or 3 sourceres to lower the risk of uncomfortable silences)

I think it will be the first time we do it 😁
But that's not a reason to avoid it, so we could try and see how does it work.

Reply:
I think it's a nice idea but choosing correctly who is going to go. I'm sorry but it's true that the lack of social abilities could be an issue. 
I agree with David, only one person could be a bit awkward. 2 or 3 would be better. No more! I guess the goal of that it is to create a link with the new employee. If there are more than 3 people, I think it's difficult to create a trust relation. 

I don't know if we should try to do it on Monday. We have 2 new employees but one of them already has friends at the office. In this case, I think it's not worth it to do it.

Reply:
yeah probably too late to organize for Monday, I think you're right maximum 2 or 3

Reply:
@bluer73 I think we could merge this once the mention about the lunch is changed from one to 2 or 3.